spec/factories/internal_post.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :internal_post do
+    user
+    title 'Example Discussion'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.